roles/slurm/tasks/controller.yml

```diff
@@ -39,6 +39,7 @@
     target: '/lib64(/.*)?'
     setype: mysqld_db_t
     state: present
+    ignore_selinux_state: yes
```

Comment:
Ignoring the selinux state seems like a bad idea. The Ansible docs indicate this is supposed to be used in scenarios like a `chroot` environment where we can't get the state in the first place. Why are we doing this here?

Reply:
I forgot why I had to set this. Maybe it was also for a clients setup that refused to work unless this setting was added. I can remove this.

Reply:
I'll remove this too.

Reply:
Fixed in commit 42a8792